Control/public/Model.js

```diff
@@ -86,6 +88,34 @@ export default class Model extends Observable {
     this.init();
   }
 
+  /** 
+   * Sets user role
+   * @returns {object} User's role 
+   */ 
+  setRole() {
+    let role = ROLES.Guest;
+    if (this.session.access.some(role => role.startsWith(PREFIX.SSO_DET_ROLE))) {
+      role = ROLES.Detector;
+    }
+    if (this.session.access.includes('global')) {
+      role = ROLES.Global;
+    }
+    if (this.session.access.includes('admin')) {
+      role = ROLES.Admin;
+    }
+    return role;
+  }
+
+  /** 
+   * Evaluate whether action is allowed for given role
+   * @param {ROLES} target role
```

Comment:
I believe ESLINT will complain a bit here. The standard for documenting a parameter includes the parameter name
Something like
```
 * @param {ROLES} role - target role
 ```